### PR TITLE
Add support for inner listing of addresses in event log

### PR DESCRIPTION
**Pull request by hansl**
This requires that sub-types of multisig (anything types that are used in fields with an inner tag) implement a trait, so I added a tag for Multisig arguments that defaults to empty set instead if the trait is not implemented. Some types would never implement it anyway.

I re-enabled the test that was disabled, and added a new test for a level 2 recursion. This should be enough to tell us it covers ALL recursive event logs.

Test coverage could be improved but if it builds it works.

### CODE REVIEW

File: src/many-modules/src/_4_events.rs [hunk 1]
```diff
@@ -757,19 +801,20 @@ impl EventLog {
...
         self.content.symbol()
     }
 
-    pub fn is_about(&self, id: &Address) -> bool {
+    pub fn is_about(&self, id: Address) -> bool {
         self.content.is_about(id)
     }
 }
 
 #[cfg(test)]
```
Comment (fmorency) [inline src/many-modules/src/_4_events.rs:813]: Can you add a test covering all event types with addresses?

File: src/many-modules/src/_4_events.rs [hunk 2]
```diff
@@ -558,26 +581,47 @@ macro_rules! encode_event_info {
...
             $( $( $name($arg), )? )*
         }
 
         impl AccountMultisigTransaction {
-            pub fn symbol(&self) -> Option<&Address> {
+            pub fn symbol(&self) -> Option<Address> {
                 // TODO: implement this for recursively checking if inner infos
                 // has a symbol defined.
                 None
             }
```
Comment (fmorency) [inline src/many-modules/src/_4_events.rs:606]: Is this even used anymore?

**Review by fmorency — CHANGES_REQUESTED** [review]
Thanks!

nit: better comment the macros.

File: src/many-modules/src/_4_events.rs (hunk 2 above)
Comment (hansl) [reply src/many-modules/src/_4_events.rs:670]: Nope, removed.

**Review by fmorency — APPROVED** [review]

**Pull request merged**